dev/Materials/Acrylic/AcrylicBrush_19h1_themeresources.xaml

```diff
@@ -26,23 +26,32 @@
                 x:Key="AcrylicBackgroundFillColorDefaultInverseBrush"
                 TintColor="#FCFCFC"
                 TintOpacity="0.0"
-                FallbackColor="#FCFCFC"
+                FallbackColor="#F9F9F9"
                 BackgroundSource="HostBackdrop"/>
             <local:AcrylicBrush
                 x:Key="AcrylicBackgroundFillColorBaseBrush"
                 TintColor="#202020"
                 TintOpacity="0.0"
                 FallbackColor="#202020"
                 BackgroundSource="HostBackdrop"/>
-            <SolidColorBrush x:Key="SystemControlDefaultBrighteningBrush" Color="#FFFFFF" Opacity="0.0419"/>
+            <local:AcrylicBrush x:Key="AccentAcrylicBackgroundFillColorDefaultBrush"
+                TintColor="{ThemeResource SystemAccentColorDark1}"
+                TintOpacity="0.8"
+                FallbackColor="{ThemeResource SystemAccentColorDark1}"
+                BackgroundSource="HostBackdrop"/>
+            <local:AcrylicBrush x:Key="AccentAcrylicBackgroundFillColorBaseBrush"
+                TintColor="{ThemeResource SystemAccentColorDark2}"
+                TintOpacity="0.8"
+                FallbackColor="{ThemeResource SystemAccentColorDark2}"
+                BackgroundSource="HostBackdrop"/>
         </ResourceDictionary>
 
         <ResourceDictionary x:Key="Light">
             <local:AcrylicBrush
                 x:Key="AcrylicBackgroundFillColorDefaultBrush"
                 TintColor="#FCFCFC"
                 TintOpacity="0.0"
-                FallbackColor="#FCFCFC"
+                FallbackColor="#F9F9F9"
                 BackgroundSource="HostBackdrop"/>
             <local:AcrylicBrush
```

Comment:
FallbackColor="#F9F9F9"

Reply:
But the value already is #F9F9F9 😅

Reply:
For AcrylicInAppFillColorDefaultBrush

AcrylicBackgroundFillColorDefaultBrush and AcrylicInAppFillColorDefaultBrush have identical values

Reply:
Ah okay, updated now.